### DISCUSSION TITLE
Trying to build a PDF

### DISCUSSION THREAD

**Discussion by stephaniehanor** [thegetty/quire#208] [Get Help] [answered]
I'm getting the following error when I try to build a pdf. I've tried reinstalling Prince, but that didn't do anything. Any suggestions?

Thanks!

mac-974:unseen shanor$ quire pdf
⠏ Building PDFBrowserslist: caniuse-lite is outdated. Please run next command `npm update`
⠋ Building PDFprince: http://localhost:1313/essay%201: error: The requested URL returned error: 404 Not Found
prince: http://localhost:1313/essay%201: error: could not load input file
prince: error: failed to load all input documents
✖ Error: Command failed: prince http://localhost:1313/ http://localhost:1313/contents http://localhost:1313/intro http://localhost:1313/essay%201 http://localhost:1313/essay%202 http://localhost:1313/essay%203 http://localhost:1313/essay%204 http://localhost:1313/catalogue http://localhost:1313/catalogue/1 http://localhost:1313/catalogue/2 http://localhost:1313/catalogue/3 http://localhost:1313/catalogue/4 http://localhost:1313/catalogue/5 http://localhost:1313/catalogue/6 http://localhost:1313/catalogue/7 http://localhost:1313/catalogue/8 http://localhost:1313/catalogue/9 http://localhost:1313/catalogue/10 http://localhost:1313/catalogue/11 http://localhost:1313/catalogue/12 http://localhost:1313/catalogue/13 http://localhost:1313/catalogue/14 http://localhost:1313/catalogue/15 http://localhost:1313/catalogue/16 http://localhost:1313/catalogue/17 http://localhost:1313/catalogue/18 http://localhost:1313/catalogue/19 http://localhost:1313/catalogue/20 http://localhost:1313/catalogue/21 http://localhost:1313/catalogue/22 http://localhost:1313/catalogue/23 http://localhost:1313/catalogue/24 http://localhost:1313/catalogue/25 http://localhost:1313/catalogue/26 http://localhost:1313/catalogue/27 http://localhost:1313/catalogue/28 http://localhost:1313/catalogue/29 http://localhost:1313/catalogue/30 http://localhost:1313/catalogue/31 http://localhost:1313/catalogue/32 -o static/downloads/output.pdf --no-warn-css-unsupported --no-warn-css-unknown --no-warn-css
prince: http://localhost:1313/essay%201: error: The requested URL returned error: 404 Not Found
prince: http://localhost:1313/essay%201: error: could not load input file
prince: error: failed to load all input documents

**Comment by geealbers** [comment, marked as answer]
Hi @stephaniehanor, it look like you have some files called `essay 1.md`, `essay 2.md`, etc? With spaces in them, or maybe it’s some other special character? Those are causing some errors with the resulting URLs. If you replace the spaces or special characters in those file names with hyphens or something like that, the PDF generation should work for you. Let us know!

**Comment by stephaniehanor** [comment]
Thanks, that did it! Love it when it's something simple.



On Mon, Nov 15, 2021 at 3:17 PM Greg Albers ***@***.***>
wrote:

> Hi @stephaniehanor <https://github.com/stephaniehanor>, it look like you
> have some files called essay 1.md, essay 2.md, etc? With spaces in them,
> or maybe it’s some other special character? Those are causing some errors
> with the resulting URLs. If you replace the spaces or special characters in
> those file names with hyphens or something like that, the PDF generation
> should work for you. Let us know!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/thegetty/quire/discussions/208#discussioncomment-1647878>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANZDL4HJ4O3IDGD5D6B6KYLUMGIIFANCNFSM5IC225YA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Dr. Stephanie Hanor
Director
Mills College Art Museum
5000 MacArthur Blvd.
Oakland, CA 94613
619-961-5161 cell
mcam.mills.edu

-- 

Sign up for our MAILING LIST <https://mcam.mills.edu/latest/> and follow us
on FACEBOOK <https://www.facebook.com/MillsArtMuseum> + INSTAGRAM
<https://www.instagram.com/millsartmuseum/>+ TWITTER
<https://twitter.com/MillsArtMuseum>

Read MCAM's archive & collections blog GLASS CUBE
<https://medium.com/glass-cube>
